app/controllers/staff/activity_forms_controller.rb

```diff
@@ -272,6 +275,18 @@ def flow
     params.require(:activity).permit(:flow).fetch("flow", nil)
   end
 
+  def sdg_1
+    params.require(:activity).permit(:sdg_1).fetch("sdg_1", nil)
+  end
+
+  def sdg_2
+    params.require(:activity).permit(:sdg_2).fetch("sdg_2", nil)
+  end
+
+  def sdg_3
+    params.require(:activity).permit(:sdg_3).fetch("sdg_3", nil)
+  end
```

Comment:
I dunno what you think, but I'm wondering if it might be better if we fetched all the sustainable development goals in one go, rather than having them as seperate parameters. I know it's a bit of a departure from what we've done elsewhere, but as they're all being collected on the one page, I don't think it's a huge deal. Would help keep this controller from getting too long and bloated. Might be worth actually passing the param fetching stuff to its own class at some point. Anyway, just a brain dump

Reply:
I actually went back and forth on this a few times - did this for consistency, but actually these three fields are related, so make sense having the params being permitted through one method.

Reply:
Yeah, I think there's a bigger piece of work to do around getting all this into an ActivityUpdater class of its own, which I've already got some ideas about. Let's leave it as it is now, in the interests of getting it out.